ssn/rdf/examples/timeseries-result-inline.ttl

```diff
@@ -28,17 +28,41 @@ ex:ts159i
   sosa:hasSimpleResult """2017-04-15T20:00:00+00:00,3.24
   2017-04-15T20:01:00+00:00,3.21
   2017-04-15T20:02:00+00:00,3.15
-  2017-04-15T20:03:00+00:00,3.15"""^^ex:CSV ;
+  2017-04-15T20:03:00+00:00,3.15"""^^ex:SpeedObservationCSVRecord ;
   rdfs:comment """The result of the observation is encoded as a CSV literal.  
   In this example, the CSV has four rows each representing a member of the time-series.  
   Each member value is composed of a timestamp, and a quantity in metres per second."""@en ;
 .
-ex:CSV a rdfs:DataType ;
-  skos:definition """An N-D vector, each row representing a 
-  member composed of N comma-separated values."""@en ;
-  rdfs:comment """This datatype is an informative example only, 
-  and is not part of the SSN Ontology."""@en ;
-.
 ex:station223 a sosa:FeatureOfInterest .
 ex:p1 a sosa:Property .
-ex:fooglemeter39 a sosa:Sensor .
\ No newline at end of file
+ex:fooglemeter39 a sosa:Sensor .
```

Comment:
In my view this is a challenging example. It took me several minutes to figure what was happening here, I wonder how someone learning SOSA/SSN may struggle.

If you wish to represent a series of observations as a simple result, it would be better to use some existing Datatype. Otherwise, if you really wish to go forwards with an SWE Datatype then it should be specified correctly with a regular expression restricting `xsd:string`. See the example in the spec: https://www.w3.org/TR/owl2-syntax/#Datatype_Definitions

Reply:
This was discussed at some length in the telecons on 2025-03-19, 2025-02-05. 
The issue is that there is no 'existing' Datatype for a specific CSV format with designated fields. 

In fact this is precisely the problem that SWE-Common addresses - it provides a schema language for compact datasets represented as extended strings. 
And since SWE-Common is an OGC standard, then it is appropriate for us to re-use it here. 

Even if it is possible to use an `xsd:pattern` to define a lexical form for a CSV record, this still would not associate the fields with semantics in the way that the SWE Common schema does.

Reply:
Meanwhile I've added a linefeed to make it easier to read, and fixed the syntax error on the `rdf:JSON` datatype - 1e7a0a62c1a3bec7a2f0789e32a58567d48c0d3a